tcks/apis/websocket/pom.xml

```diff
@@ -19,16 +19,35 @@
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-        <groupId>jakarta.tck</groupId>
-        <artifactId>websocket-tck</artifactId>
-        <version>2.2.0</version>
+        <groupId>org.eclipse.ee4j</groupId>
+        <artifactId>project</artifactId>
+        <version>2.0.0-M1</version>
+        <relativePath/>
     </parent>
 
+    <groupId>jakarta.tck</groupId>
     <artifactId>websocket-tck-platform-tests</artifactId>
+    <version>11.0.2-SNAPSHOT</version>
     <packaging>jar</packaging>
+
     <name>Jakarta Websocket TCK Platform Tests</name>
-    <version>11.0.2-SNAPSHOT</version>
-    
+    <description>Jakarta WebSocket Platform TCK Tests</description>
```

Comment:
The description may not be accurate.  Need to research the status of these WebSocket tests a bit before merging this change.